NitroxClient/MonoBehaviours/MultiplayerVehicleControl.cs

```diff
@@ -71,15 +80,20 @@ internal virtual void Exit()
 
     public abstract class MultiplayerVehicleControl<T> : MultiplayerVehicleControl
     {
+
         private readonly FieldInfo steeringWheelYaw = ReflectionHelper.GetField<T>("steeringWheelYaw");
         private readonly FieldInfo steeringWheelPitch = ReflectionHelper.GetField<T>("steeringWheelPitch");
+
         protected T SteeringControl;
+        protected T ArmPositions;
```

Comment:
This is specific to the exosuit - should it be in the generic MultiplayerVehicleControl class?